.github/workflows/review-release-notes.yml

```diff
@@ -23,20 +21,25 @@ jobs:
           files: |
             Packs/**/ReleaseNotes/**
           since_last_remote_commit: false
-
-      - name: Install SDK
+      - name: Setup Python
```

Comment:
streamline the SDK installation thru poetry, this is the newer standard used in content.